README.adoc

```diff
@@ -132,14 +137,43 @@ System.out.println("${'$'}{a+b}");
 the result is : `${a+b}`
 
 
-## How it works
+## How to control the generated code
 
 Better Strings is a Java Annotation Processor,
 but it does not process specific annotations, it makes AST modification of your code while javac compiling it.
 
-NOTE: Keep in mind that this feature should be used carefully.
-You shouldn't write too much code inside string literals
-because it is too difficult to maintain and maybe not obvious for debugging.
+By default, each `${...}` occurrence translates into an invocation of `String#valueOf`.
+For instance, a string:
+
+[source, java]
+----
+"Result: ${obj}.method() = ${obj.method()}"
+----
+
+will yield:
+
+[source, java]
+----
+"Result: "
+  + String.valueOf(obj)
+  + ".method() = "
+  = String.valueOf(obj.method())
```

Comment:
Instead of `=` it should be `+` here... to be fixed with the next release

Reply:
wow, I read this text but didn't notice a typo =)
I have fixed it now.